application-admintools-ui/src/main/resources/AdminTools/Code/AdminToolsJS.xml

```diff
@@ -271,6 +279,22 @@
       }
     }
   }
+
+  $(document).on('click', '#confirmCacheFlushModal .btn-primary', function(event) {
+    event.preventDefault();
+    const flushButton = $('#confirmCacheFlushModal .btn-primary');
+    const target = flushButton.attr('data-action');
+    $.post(target)
+    .done(function() {
```

Comment:
Indentation is a little strange, but it might be a personal preference. So, just a suggestion:
```
$.post(target).done(function() {
  ...
}).fail(function(error) {
  ..
});
```